CHANGELOG.md

```diff
@@ -917,7 +982,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fix README
 
 
-[unreleased]: https://github.com/Dasharo/meta-dts/compare/v1.2.23..HEAD
+[2.1.0]: https://github.com/Dasharo/meta-dts/compare/v2.0.1-rc3..v2.1.0
+[2.0.1-rc3]: https://github.com/Dasharo/meta-dts/compare/v2.0.1-rc2..v2.0.1-rc3
+[2.0.1-rc2]: https://github.com/Dasharo/meta-dts/compare/v2.0.1-rc1..v2.0.1-rc2
+[2.0.1-rc1]: https://github.com/Dasharo/meta-dts/compare/v2.0.0..v2.0.1-rc1
+[2.0.0]: https://github.com/Dasharo/meta-dts/compare/v2.0.0-rc6..v2.0.0
```

Comment:
Why from `rc6` and not `rc7`?

Reply:
fixed https://github.com/Dasharo/meta-dts/compare/3ed77b24899849f939d8bf1a173027837e63bed4..0795508d026a009491115788cbc4fa0e560309a8